distutils/_msvccompiler.py

```diff
@@ -224,7 +233,10 @@ def initialize(self, plat_name=None):
 
         self._paths = vc_env.get('path', '')
         paths = self._paths.split(os.pathsep)
-        self.cc = _find_exe("cl.exe", paths)
+        if self.use_clang_cl:
```

Comment:
I don't see that this value is set anywhere. When/how is this property set?

Reply:
Oops. Fixed now